docs/admin/installation/set_up_transfer_and_export_device.rst

```diff
@@ -199,6 +199,15 @@ you need to install the VeraCrypt software. The `guide by Freedom of the Press
 Foundation <https://freedom.press/training/encryption-toolkit-media-makers-veracrypt-guide/>`__
 provides instructions for encrypting storage media using VeraCrypt.
 
+.. Remove the following warning once securedrop-docs#599 and
+   veracrypt/VeraCrypt#1422 are resolved.
+
+.. warning::
+
+   If you plan to use your *Export Device* with one or more computers running
```

Comment:
I like this language in general, but "one or more computers" sounds a bit unusual to me.

Would "any computers" be better?

Reply:
Good call.  Simplified even further in 30f7600577ab1c1604a0c9ed6a064bff41e1ba59.